### ISSUE TITLE
Bug: Neither oauth nor personal access token work, even on public repos

### ISSUE THREAD

**Issue by Nezteb** [EnixCoda/Gitako#178]
On Chrome 93.0.4577.82

Using Gitako on the Gitako repo:
<img width="751" alt="Screen Shot 2021-09-16 at 11 02 17 AM" src="https://user-images.githubusercontent.com/3588798/133646442-c0d23e2f-4136-4e86-aa20-29281edbc411.png">

My oauth setup:
<img width="1069" alt="Screen Shot 2021-09-16 at 11 02 25 AM" src="https://user-images.githubusercontent.com/3588798/133646483-a0c6c2aa-b134-4afa-b469-a5cc4d81d504.png">

Not sure what else to do. I tried a personal access token as well to no avail.

**Comment by EnixCoda** [comment]
Thank you for submitting this issue @Nezteb 

But I cannot reproduce it. It works for me and few other users with whom I tried to reproduce this issue.

Let's keep it open to see if there are any other users also having this issue.

**Comment by Nezteb** [comment]
I resolved this! I had to explicitly toggle "Enable Gitako on this domain" within the Chrome extension menu. The machine I'm on is a fresh macOS install with a fresh Chrome profile, so I wonder if this is some new default for Chrome extensions? Although every other extension I've installed doesn't require this step. If more users report this in the future, that might be it.

<img width="395" alt="Screen Shot 2021-09-23 at 3 23 37 PM" src="https://user-images.githubusercontent.com/3588798/134578762-72550bf5-987f-4af0-8bef-a34f39434b61.png">
